docs/Quantiles.ipynb

```diff
@@ -0,0 +1,562 @@
+              "│ i64          ┆ i64          ┆ i64          ┆ i64          │\n",
+              "╞══════════════╪══════════════╪══════════════╪══════════════╡\n",
+              "│ 20           ┆ 32           ┆ 58           ┆ 72           │\n",
+              "└──────────────┴──────────────┴──────────────┴──────────────┘"
+            ]
+          },
+          "execution_count": 17,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "multiple_quantiles = context.query().select(\n",
+        "    [pl.col(\"AGE\").fill_null(mean_age).dp.quantile(q, candidates).alias(f\"Quantile_{q}\") for q in [0.2, 0.4, 0.6, 0.8]]\n",
+        ").release().collect()\n",
+        "multiple_quantiles"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "fYAFhv6vXVbO"
+      },
+      "source": [
+        "Here we computed some sample quantiles and the differentially private values are relatively close to the actual values. Notice that we also specified a different alias for each column. Polars uses the original column name on derived columns, so aliases are needed to distinguish these columns.\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "zA0D8TeJXVbO"
+      },
+      "source": [
+        "## Breaking Down the Quantile Method"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "TyHOsKIiXVbO"
+      },
+      "source": [
+        "Now that you know how to use the quantile method, we can also break it down for you a little more.\n",
+        "\n",
+        "You might also wonder why there are multiple steps to the quantile method. Why don't we just add noise directly?\n",
+        "\n",
+        "Directly adding noise can change a particular point and alter the sensitivity (max - min) greatly, so constructing a differentially private quantile requires more sophistication."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Y2bdy-0SXVbO"
+      },
+      "source": [
+        "### 1. Compute the Discrete Quantile Score for Each Candidate"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "6XSkx3kiXVbO"
+      },
+      "source": [
+        "The `discrete_quantile_score` function takes in the same parameters as the quantile function. It computes a utility score for each candidate quantile that represents how close each candidate is to the true quantile. Lower scores are closer to 0 are more accurate."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 33,
+      "metadata": {},
+      "outputs": [
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "/var/folders/dg/nlwnz_jj7rsblr2tzwj962b40000gn/T/ipykernel_50483/2347966570.py:7: RuntimeWarning: invalid value encountered in cast\n",
```

Comment:
Would like the code to be able to run without warnings... but if it's unclear to you, make it a TODO and we can sort it out later.